week0/homework/person.js

```diff
@@ -0,0 +1,29 @@
+
+ exports.person=class person 
```

Comment:
Use whitespace around the operators, and Title case for Classes, so like: `exports.Person = class Person {`
Good resource on JS writing style: http://javascript.crockford.com/code.html (like see http://javascript.crockford.com/code.html#whitespace)